### PR TITLE
Integration testing readme

### PR DESCRIPTION
**Pull request by baude**
Wrote a small primer to setup for integration testing.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: README.ginkgo.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The following instructions assume your GOPATH is ~/go. Adjust as needed for your
+environment.
+
+### Installing ginko
+Fetch and build ginkgo with the following command:
+```
+GOPATH=~/go go get -u github.com/onsi/ginkgo/ginkgo
+```
+No install the ginkgo binary into your path:
```
Comment (TomSweeneyRedHat) [inline README.ginkgo.md]: "No install" to "Now install"?

Comment (baude) [reply README.ginkgo.md]: fixed

**Comment by baude** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
I am fine with this. Although I think we should add the install of ginko to the Makefile.  That should be done in a separate PR.

@rh-atomic-bot r+

**Pull request closed without merging**